### DISCUSSION TITLE
Slideshows

### DISCUSSION THREAD

**Discussion by ajsteele** [ente/ente#603] [Enhancements]
It would be great to have the ability to display photos in a slideshow.

Simple implementation: just a play button in an album/the All Photos view, with a shuffle checkbox to randomise order or not.

Dream implementation: ability to define custom slideshows by selecting or excluding particular albums, eg 'Europe trip 2022' slideshow might include the albums 'France', 'Germany' and 'Spain' folders, etc.

Thanks for a great app!

**Comment by abhinavkgrd** [comment]
Thank you @ajsteele, This sounds like a really nice feature, have added it to the roadmap 😄 .

Will get it shipped soon

**Reply by oaklandnative** [reply]
Are there any updates on a native slideshow? Is it actively being worked on? 

My wife and I have a shared Google Photos album that we run as a shuffled slideshow 24/7 on a tablet. When either of us add new photos, they automatically get shuffled into the slideshow. The lack of any remotely similar feature is really the only thing that is keeping us from switching to Ente. Seems strange that casting is the only option, and also that there are no basic controls like shuffle, photo duration time, or even skip forward or back.

**Comment by ajsteele** [comment]
Thanks for the absurdly rapid reply, can't wait!

**Comment by mnvr** [comment]
We'll soon be shipping support for cast. While not exactly what this feature request is about, there is an intersection of use cases which'll get solved by casting on a separate browser window to let it run through an unattended slideshow (doesn't need extra hardware). Just mentioning this in case it helps someone's usecase.

**Reply by mrmurphy** [reply]
Nice! I noticed the cast feature and came back here to see if slideshows in-app are still planned. That'd be my preferred method.

**Reply by spaasis** [reply]
Yeah, casting is nice for auto playing on the background, but for a slideshow I would like to have the option of speeding up the flow, going back and staying at the previous photo e.g. If these could be controller via the app, that's fine, but basic controls for the web ui would be my preference as well

**Reply by jinwater88** [reply]
Is the memory function in iPhone photo album the same as the slide show here? What technology is used to achieve it? Moviepy?

**Reply by ph00lt0** [reply]
AppleTV support with cast works well for this, although photos do not seem ordered. Besides that it would be great to have slideshow option on tablets.

**Reply by alnetloc** [reply]
Any ability to suppress captions while running slideshow?  That would be great as the text, while useful at times, clutters the image if you run image loop or you are narrating a slideshow.

**Comment by lnkd61** [comment]
New to Ente and was looking for a way to do exactly this on my Pixel Tablet to replace the Google Photos slideshow feature. It would be nice if Ente provided an Android Screensaver feature that lets you pick which albums to display a slideshow of. That way it would work as a drop-in replacement and it'll work on other Android based devices like Android TV boxes.

**Comment by Ansago** [comment]
Hi Ente team, just wanted to throw my hat in the ring for this request as well! My wife and I love your app and it's helped us get away from Google Photos, but we do still use it for our at-home display. If Ente offered the ability to play a local slideshow that would be our last need for ditching Google entirely.

**Reply by mnvr** [reply]
Just to get more data points - will the Android Screensaver feature work for the device where you'd like to see the photos? Or your use case needs an in-app full-screen slideshow - so you're expecting to open the Ente app itself on your viewing device, and then go to the album of your choice and launch the slide show.

Thanks for the comments!

**Reply by Ansago** [reply]
Thank you for responding! Ideally we would be able to open the Ente app itself on the device and run a full-screen slideshow from within the app. That way we can both update the slideshow album from our other devices (using Ente) and don't have to export images to the local device playing the slideshow.

**Reply by lnkd61** [reply]
> Just to get more data points - will the Android Screensaver feature work for the device where you'd like to see the photos? Or your use case needs an in-app full-screen slideshow - so you're expecting to open the Ente app itself on your viewing device, and then go to the album of your choice and launch the slide show.
> 
> Thanks for the comments!

For me, I would like to be able to pick Ente as an option in Android's Screen Saver settings. That way, when I dock my Pixel tablet, it'll automatically start a slideshow of photos from my album. Same thing with my Phone while charging or when my Google TV boxes go into screen saver mode.

**Reply by alnetloc** [reply]
Another pitch to depress (or not) captions for any slideshow.

**Comment by ghost** [comment]
Hi Ente team, adding a vote for this feature as its the thing my family wants the most added to the Android and iPhone apps. Starting a slideshow on the host device with custom options, and the ability to loop indefinitely. Options display pics with certain people, locations, time periods etc would be a cool bonus. Thank you!

**Comment by EdgeCon** [comment]
Throwing my hat in these votes too! I'd love to turn my old Android tablet into a digital picture frame! I love knowing my photos are safe and private, so this would be a great gift for family members that keep bugging me to send photos to their sketchy digital picture frame they bought off Amazon :).

**Comment by aChrisYouKnow** [comment]
Another +1 for this feature. Looking for a way to buy someone a digital frame and put a slideshow of an ente album which already has a public sharing link onto the frame in the form of a slideshow.

**Comment by TolikTentser** [comment]
Also +1 for slideshow. I think it is functional and not so complex technically

Thank you

**Comment by Ansago** [comment]
Thanks so much for adding this in the latest update! It’s a great feature to have and helps keep us in the Ente ecosystem. My only other request would be to add longer durations (1 min, 5 min, 10 min).

**Comment by matclab** [comment]
The new feature is very nice ! Thank you.

A nice addition would be that the application memorizes its running status. 
If it was closed while showing a slideshow for albums X, Y and Z, then when launched, it starts back to showing the slideshow for those albums (It would provide the feature : slideshow on a tablet as a photoframe, where ente is launched at boot).

**Comment by matclab** [comment]
Two more features would be nice in order to replace my existing (non ente)  slideshow app would be :
- allow random non repeating play accross all images from all selected albums
- when screen is touched, allow to swipe back or forth to the previous or next images. Either then pause the slideshow until a play button is pressed.
